src/pages/home/Organizers.tsx

```diff
@@ -1,64 +1,68 @@
 import * as React from "react";
-import donavon2 from "../../assets/donavon@2x.png";
-import harry2 from "../../assets/harry@2x.png";
-import jenn2 from "../../assets/jenn@2x.png";
-import matthew2 from "../../assets/matthew@2x.png";
+import { shuffle } from "lodash";
+import donavon from "../../assets/donavon@2x.png";
+import harry from "../../assets/harry@2x.png";
+import jenn from "../../assets/jenn@2x.png";
+import matthew from "../../assets/matthew@2x.png";
 import Section from "../../components/Section";
 import TextGroup from "../../components/TextGroup";
 
+type Organizer = {
+  twitter: string;
+  image: string;
+  name: string;
+};
+const Organizers: Organizer[] = shuffle([
```

Comment:
forgive the stupid question 'cuz I don't know Gatsby, but won't this shuffle on each _build_ not on each page refresh?

Reply:
It ends up shuffling on each page load when react mounts

Reply:
i thought gatsby was a *static* site generator. if so, it _should_ produce static html server side at build time. therefore, there is no react and component mount on the client to shuffle.

if not, how is this any different than a standard react app?